tests-ui/tests/widgets/dynamicCombo.test.ts

```diff
@@ -0,0 +1,90 @@
+import { createPinia, setActivePinia } from 'pinia'
+import { describe, expect, test } from 'vitest'
+import { LGraphNode } from '@/lib/litegraph/src/litegraph'
+import { transformInputSpecV1ToV2 } from '@/schemas/nodeDef/migration'
+import type { InputSpec } from '@/schemas/nodeDefSchema'
+import { useLitegraphService } from '@/services/litegraphService'
+
+setActivePinia(createPinia())
```

Comment:
Could you use a [testing pinia](https://pinia.vuejs.org/api/@pinia/testing/functions/createTestingPinia.html)?